### ISSUE TITLE
cli: netdataregister command should return Done

### ISSUE THREAD

**Issue by turon** [openthread/openthread#6]
Running the netdataregister doesn't return the expected 'Done' response:

```
./soc --nodeid=1 -S
netdataregister
netdataregister
```

After running the command, other commands still work.  But if the command is run twice, the shell becomes unresponsive to all commands:

```
./soc --nodeid=1 -S
netdataregister
netdataregister
panid
panid
65535
Done
netdataregister
netdataregister
panid
panid
```

**Comment by jwhui** [comment]
I submitted #16 to address the hang issue.

"Done" is only printed if the command completes successfully.  In your example, OpenThread was not started yet, so netdataregister is not successful.  Should we enhance the CLI to print "Error" with the error code instead?

**Comment by turon** [comment]
Yes, "Error: <code or optional details>\r\n" would be a good format and deterministically parseable by the testing suite.

**Comment by jwhui** [comment]
Fixed in #66.
